content/ja/docs/contributing/pull-requests.md

```diff
@@ -0,0 +1,467 @@
+---
+title: コンテンツの提出
+description: GitHub UI またはローカルのフォークを利用して、新しいコンテンツまたはコンテンツの変更を提出する方法を学びます
+aliases: [new-content]
+weight: 15
+default_lang_commit: 95b782b83ea5491bf366237260427b989ac7f6c8
+---
+
+新しいドキュメントの内容を追加したり、既存のコンテンツの改善をするには、[プルリクエスト][PR] （PR）を提出してください。
+
+- もし変更が小さかったり、自身が Git に慣れていなかったら、[GitHub を使用した変更](#changes-using-github)を参照しページの編集方法を学んでください。
+- もし変更が大きければ、[ローカルのフォークから作業する](#fork-the-repo)を参照し、ローカルのコンピュータから変更を行う方法を学んでください。
+
+{{% alert title="ヒント: Draft ステータス" %}}
+
+コンテンツがレビューの準備ができていないことをメンテナーに知らせるために、プルリクエストのステータスを **Draft** にしてください。
+メンテナーは、Draft ステータスを解除するまでコンテンツを完全にレビューしませんが、コメントや高レベルのレビューを行うことがあります。
+
+{{% /alert %}}
+
+以下の図は新しいドキュメントにコントリビュートする方法を示しています。
+
+```mermaid
+flowchart LR
+    subgraph first[コントリビュートする方法]
+    direction TB
+       T[ ] -.-
+       B[GitHub でフォークする] --- C[マークダウンでドキュメントを作成する<br>Hugo でサイトをビルドする]
+       C --- D[フォーク先にソースコードを push する]
+       D --- E[プルリクエストを公開する]
+       E --- F[CNCF CLA に署名する]
+    end
+
+classDef grey fill:#dddddd,stroke:#ffffff,stroke-width:px,color:#000000, font-size:15px;
+classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:bold
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class A,B,C,D,E,F,G,H grey
+class S,T spacewhite
+class first,second white
+```
+
+_図 1. 新しいコンテンツにコントリビュートする。_
+
+## GitHub を使用した変更 {#changes-using-github}
+
+もし、Git のワークフローをあまり慣れていなければ、プルリクエストを作成して提出するより簡単な方法があります。
+図 2 は手順の概要を示しており、詳細はこの後に続きます。
+
+{{% alert title="Contributor License Agreement (CLA)" color=warning %}}
+
+すべてのコントリビューターは、変更が承認されてマージされる前に、[コントリビューター ライセンス契約 (CLA)][CLA] に署名する必要があります。
+
+[CLA]: ../prerequisites/#cla
+
+{{% /alert %}}
+
+```mermaid
+flowchart LR
+A([fa:fa-user 新しい<br>コントリビューター]) --- id1[(open-telemetry/opentelemetry.io<br>GitHub)]
+subgraph tasks[GitHub を使用した変更]
+direction TB
+    0[ ] -.-
+    1[1\. ページを編集] --> 2[2\. GitHub マークダウンエディターを利用して変更を加える]
```

Comment:
Markdown

Reply:
Thank you. I fixed a bellow commit.

https://github.com/open-telemetry/opentelemetry.io/pull/6197/commits/f679313feffe2a7bf29ba390c5440bdb4a5ec652